extensions/binding/org.eclipse.smarthome.binding.hue/src/main/java/org/eclipse/smarthome/binding/hue/handler/HueBridgeHandler.java

```diff
@@ -491,4 +496,19 @@ private boolean isEqual(State state1, State state2) {
         return colorModeIsEqual && effectIsEqual;
     }
 
+    @Override
+    public Collection<ConfigStatusMessage> getConfigStatus() {
+        // The bridge IP address to be used for checks
+        final String bridgeIpAddress = (String) getThing().getConfiguration().get(HOST);
+
+        // Check whether an IP address is provided
+        if (bridgeIpAddress == null) {
+
+            return ImmutableList.of(ConfigStatusMessage.Builder.error(HOST)
+                    .withMessageKeySuffix(HueConfigStatusMessage.IP_ADDRESS_MISSING.getMessageKey()).withArguments(HOST)
+                    .build());
+        }
```

Comment:
Fine, an else statement will be added. What is your opinion about the approach of first declaring the collection, then assigning it inside the if/else statement and finally using a single return? 

```java
Collection<ConfigStatusMessage> configstatusMessages;

// Check whether an IP address is provided
if (bridgeIpAddress == null) {
    configstatusMessages = Collections.singletonList(ConfigStatusMessage.Builder.error(HOST)
        .withMessageKeySuffix(HueConfigStatusMessage.IP_ADDRESS_MISSING.getMessageKey())
        .withArguments(HOST).build());
} else {
     configstatusMessages = Collections.emptyList();
}

return configstatusMessages;
```

Furthermore, what do you think about checking not only if the bridge IP address is null, but also doing `bridgeIpAddress.trim().isEmpty()` which will result in our if-statement having the form 
`if (bridgeIpAddress == null || bridgeIpAddress.trim().isEmpty())`?

Finally, what about using a regular expression matching IPv4, such as `bridgeIpAddress.matches("^(?:[0-9]{1,3}\\.){3}[0-9]{1,3}$")`?

All in all, please share your opinion about making the scope of the check greater (IP not null, not empty and successfully filtered through the matcher) instead of just checking for null value?